discoverecs.py

```diff
@@ -290,15 +291,26 @@ def print_cache_stats(self):
             )
         )
 
+    def list_clusters(self):
+        if len(self.cluster_arns) > 0:
+            return self.cluster_arns
+        cluster_arns = []
+        clusters_pages = self.ecs_client.get_paginator("list_clusters").paginate()
+        for clusters in clusters_pages:
+            for cluster_arn in clusters["clusterArns"]:
+                cluster_arns += [cluster_arn]
+
+        print(cluster_arns)
```

Comment:
Remove this print and then I'll try to merge this when I have the time 👍🏻

Reply:
Alright. Done. Thanks for the review :)